website/src/pages/index.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from "react";
+import classnames from "classnames";
+import Layout from "@theme/Layout";
+import Link from "@docusaurus/Link";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import withBaseUrl from "@docusaurus/withBaseUrl";
+import styles from "./styles.module.css";
+
+const features = [
+  {
+    title: <>Internal Architecture</>,
+    description: (
+      <>
+        Designed to efficiently manage multiple streams of data coming from
+        different sources measuring different aspects of your app.
+      </>
+    )
+  },
+  {
+    title: <>Trace Processing</>,
+    description: (
+      <>
+        Built to parse traces into a semantic trace object that can be analyzed
+        to extract meaningful metrics.
+      </>
+    )
+  }
+];
+
+const Index = () => {
+  const context = useDocusaurusContext();
+  const { siteConfig = {} } = context;
+
+  return (
+    <Layout
+      title={`${siteConfig.title} - ${siteConfig.tagline}`}
+      description={siteConfig.tagline}
+    >
+      <header className={classnames("hero hero--primary", styles.heroBanner)}>
+        <div className="container">
+          <h1 className="hero__title">{siteConfig.title}</h1>
+          <p className="hero__subtitle">{siteConfig.tagline}</p>
+          <p className="lead">
+            Profilo is an Android library for collecting performance traces from
+            production builds of an app.
+          </p>
+          <div className={styles.buttons}>
+            <Link
+              className={classnames(
+                "button button--secondary button--lg",
+                styles.getStarted
+              )}
+              to={withBaseUrl("docs/getting-started")}
+            >
+              Get Started
+            </Link>
+          </div>
+        </div>
+      </header>
+      <main>
+        {features && features.length && (
+          <section className={styles.features}>
+            <div className="container">
+              <div className="row">
+                {features.map(({ title, description }, idx) => (
+                  <div
+                    key={idx}
+                    className={classnames("col col--6", styles.feature)}
```

Comment:
Class `feature` does not exist.

```suggestion
                    className="col col--6"
```

Reply:
done